### ISSUE TITLE
2.0.0 doesn't read stdin via pipe

### ISSUE THREAD

**Issue by skatsuta** [monochromegane/the_platinum_searcher#113]
Hi, congrats on 2.0.0 release! I found a bug in it. While 1.7.8 can read stdin via pipe properly, 2.0.0 can't.

```sh
$ pt --version
1.7.8

$ echo hello | pt he
:1:hello

$ brew switch pt 2.0.0
Cleaning /usr/local/Cellar/the_platinum_searcher/1.7.7
Cleaning /usr/local/Cellar/the_platinum_searcher/1.7.8
Cleaning /usr/local/Cellar/the_platinum_searcher/2.0.0
1 links created for /usr/local/Cellar/the_platinum_searcher/2.0.0

$ pt --version
pt version 2.0.0

$ echo hello | pt he
# so many file grep results that contain "he" here...

**Comment by monochromegane** [comment]
Thank you for using pt, and reporting :^)

I removed this feature (search from stream) since v2.0.0, because the feature can't detect correct encoding.
But It seems that many user need this feature. https://github.com/monochromegane/the_platinum_searcher/pull/110#issuecomment-164247505

So, I will re-implement that this month.
Please wait new version :smiley:

**Comment by skatsuta** [comment]
@monochromegane Oh, I'm sorry I haven't noticed the behaviour is intended.
I'm looking forward to the next version release!
Thank you for developing the great tool ;)

**Comment by monochromegane** [comment]
I have released v2.0.1 now. I implemented `Search Stream` feature again in the version.
Please try it :smile: 
Thank you for using pt :beers:

**Comment by skatsuta** [comment]
@monochromegane Thanks! I'm trying it :)
